### ISSUE TITLE
scraper: ministry of immigration support criterias

### ISSUE THREAD

**Issue by noamoss** [OpenBudget/BudgetKey#391]
All criteria and relevant documents are published in a single page, with a separate section per year, and scan this page once a month:

https://www.gov.il/he/Departments/publications/Call_for_bids/tmichot_mosdot_tzibur

I suggest to create an annual item with the following properties:

- title 'תמיכות במוסדות ציבור לשנת [year]'
- textual content/description: "תמיכה בפעולות חברה ותרבות המסייעות לקליטה בקהילה של עולים באמצעות סיוע כספי לארגונים הפועלים ומקדמים את תחום קליטת העלייה."
- tags: "עולים חדשים", "קהילה ורווחה"
- publisher: משרד הקליטה והעליה
- publish date: the month/year in which we find out of the udpate
- attach all the files in the section with their labels
- I would add 'לא ידוע' as due date and make sure to mark it as closed when the budget year ends. (@akariv any other idea here?)

![image](https://user-images.githubusercontent.com/3581741/62831613-0da45580-bc2b-11e9-9b4d-5971d920144a.png)

**Comment by akariv** [comment]
Following our previous discussions, aggregated items have no place in our current system and are of little value generally (as they are not discoverable and contain very little useful information).

On the other hand, it seems that the relevant information is all published as 'tenders' here:

https://www.gov.il/he/departments/publications?OfficeId=27db3169-ab0e-490c-af70-6d03133cb1f3&limit=10&publicationType=be652b8e-bb3c-4a5b-bb5d-78a37edbff82

To filter out duplicates, we can use the phrase 'מסמכי המרכז מפורסמים באתר מינהל הרכש הממשלתי'.

**Comment by noamoss** [comment]
@akariv 
>Following our previous discussions, aggregated items have no place in our current system and are of little value generally (as they are not discoverable and contain very little useful information).

We better not make a general rule, and check each source separately. This source includes some relevant content, as you said. 

> it seems that the relevant information is all published as 'tenders'...

not only as 'tenders' also as 'תמיכות ומענקים':
https://www.gov.il/he/departments/publications?OfficeId=27db3169-ab0e-490c-af70-6d03133cb1f3&limit=10&publicationType=7b76d87f-d299-4019-8637-5f1de71c9523

If we cover both, it seems to be not perfect, but good enough. Right now there are no 'קולות קוראים' published for this office, but I would like to cover this option as well, to make sure we are minimizing uncovered content from gov.il. I guess there is no problem here, right?

bottom line, as far as I can tell, we can count on gov.il for the ministry of immigration publishments as long as we cover all 3 relevant content types.

>To filter out duplicates, we can use the phrase 'מסמכי המרכז מפורסמים באתר מינהל הרכש הממשלתי'.

not sure I understand this comment. Please explain.

**Comment by akariv** [comment]
both קולות קוראים and תמיכות ומענקים are already being fetched from gov.il
regardless of the publisher so naturally that's covered.
My comment referred to מכרזים, which we're not fetching as it might
duplicate with data from mr.gov.il.

In order to avoid duplication, we can fetch מכרזים only for this publisher,
and use the phrase I mentioned to filter out items which exist in both
sources.

@noamoss can you green light this plan?

**Comment by noamoss** [comment]
agreed

**Comment by akariv** [comment]
Sample item:

https://next.obudget.org/i/calls_for_bids/call_for_bids/51100/d93361de-ca24-471e-a001-47bb99d39c72?li=9
